std/csv.d

```diff
@@ -1013,14 +1013,14 @@ public:
 
         if (!_input.range.empty)
         {
-           if (_input.range.front == '\r')
-           {
-               _input.range.popFront();
-               if (_input.range.front == '\n')
-                   _input.range.popFront();
-           }
-           else if (_input.range.front == '\n')
-               _input.range.popFront();
+            if (_input.range.front == '\r')
```

Comment:
No identation is alright:

https://github.com/BBasile/phobos/blob/82d6dbf8175055e1851bd6601dc20d0f5649e2f7/std/csv.d#L1014